src/main/scala/com/redislabs/provider/redis/redisFunctions.scala

```diff
@@ -272,6 +272,18 @@ class RedisContext(@transient val sc: SparkContext) extends Serializable {
     kvs.foreachPartition(partition => setHash(hashName, partition, ttl, redisConfig, readWriteConfig))
   }
 
+  /**
+    * @param kvs      Pair RDD of hashName/K/V
+    * @param ttl      time to live
+    */
+  def toRedisHASH2(kvs: RDD[(String, (String, String))], ttl: Int = 0)
+                 (implicit
```

Comment:
What does implicit indicate here?

Reply:
I'm sorry I don't have much knowledge for it. but It will be helpful. 
http://baddotrobot.com/blog/2015/07/03/scala-implicit-parameters/

Using implicit, even though we don't pass those parameters explicit, It will pass the params when it is compiled.

Reply:
if the compiler can't find the correct types in scope it will fail to compile.  having them marked as implicit means you don't have to explicitly call them in your code but you can if you want.